src/app/components/MediaPlayer/index.tsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable import/order */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import React, { useContext, useEffect, useRef, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { BumpType, PlayerConfig, Props } from './types.d';
+import nodeLogger from '../../lib/logger.node';
+import buildConfig from './utils/buildSettings';
+import Caption from '#app/legacy/containers/Caption';
+import getPlayerProps from '#app/legacy/containers/MediaPlayer/helpers/propsInference';
+import { RequestContext } from '#contexts/RequestContext';
+
+const logger = nodeLogger(__filename);
+
+const BumpLoader = () => (
+  <Helmet>
+    <script
+      type="text/javascript"
+      src="https://static.bbci.co.uk/frameworks/requirejs/0.13.0/sharedmodules/require.js"
+    />
+    <script type="text/javascript">
+      {`bbcRequireMap = {
+            "bump-4":"https://emp.bbci.co.uk/emp/bump-4/bump-4"
+        }
+        require({ paths: bbcRequireMap, waitSeconds: 30 });`}
+    </script>
+  </Helmet>
+);
+
+const VideoPlayer = ({ playerConfig }: { playerConfig: PlayerConfig }) => {
+  const playerElementRef = useRef();
+
+  useEffect(() => {
+    try {
+      window.requirejs(['bump-4'], (Bump: BumpType) => {
+        if (playerElementRef && playerElementRef.current && playerConfig) {
+          const mediaPlayer = Bump.player(
+            playerElementRef.current,
+            playerConfig,
+          );
+          mediaPlayer.load();
+        }
+      });
+    } catch (error) {
+      logger.error('Failed to bind SMP', error);
+    }
+  }, [playerConfig]);
+
+  return <div ref={playerElementRef} data-e2e="media-player" />;
+};
+
+const Placeholder = ({ setter }: { setter: (value: boolean) => void }) => {
+  return (
+    <button
+      type="button"
+      data-e2e="media-player__placeholder"
+      onClick={() => setter(false)}
+    >
+      TODO: CLICK TO SEE VIDEO
```

Comment:
Are we planning to replace this will real / translated placeholder text?

Reply:
yesss, as part of the ongoing work to deprecate Morph.